ehitchhiking-ui/src/app/pages/registration/registration.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "page">
+<div id="login-form">
+    <div class = "exadel-login">
+      <img src="../../../assets/images/FullColor_Word-1.png" alt="exadel-logo">
+    </div>
+
+  <fieldset class = "f-set">
```

Comment:
bad naming: absolutely unclear what is f-set

Reply:
renamed